### DISCUSSION TITLE
@egjs Flicking plugin for Vue. Not working adaptive

### DISCUSSION THREAD

**Discussion by danilpon98** [naver/egjs-flicking#694] [Q&A]
Hi, everybody. I'm using the "@egjs/flicking" plugin for a Vue 2 project. I have a slider with different panel heights. I want the height of my slider to adapt to the largest size of the panels that are in the field of view. Please tell me how I can do this?

[Live Demo](https://codesandbox.io/s/flicking-adaptive-q9bl3b?file=/src/App.vue)

**Comment by malangfox** [comment]
Hello @danilpon98.

This seems to be an error in `adaptive` option. We will notify you after this error is fixed.
Thank you for reporting this invalid behavior.

**Reply by danilpon98** [reply]
Thank you, @malangfox, I'm really looking forward to your message

**Comment by malangfox** [comment]
We have released a new version(v4.9.1) that fixes this issue.

Also, in your example, it seems that the height of each panel element inside Flicking should have a height that will change.

Before - Each panel had another panel of its own height inside it.
```ts
<flicking :options="options">
  <div v-for="(panel, i) in panels" :key="i">
    <div class="panel">{{ panel }}</div>
  </div>
</flicking>
```

After - Height of the contents in the panel are correctly applied to the panel.
```ts
<flicking :options="options" ref="flicking">
  <div class="panel" v-for="(panel, i) in panels" :key="i">
    {{ panel }}
  </div>
</flicking>
```

[You can check this similar implementation demo.](https://codesandbox.io/s/flicking-adaptive-on-vue-ri5ywl?file=/src/App.vue)

Thank you for your patience. It would be nice if this could be resolved.
